Miscellaneous Updates

VeriTas-arch opened the pull request:
- Fix mkdocs-static-i18n configurations and update README setup introductions respectively.
- Add Chinese translations.
- A GitHub Action sample workflow file is added to .github/archive, which can replace the `gh-pages` branch if it is to be simplified in the future.

See the commit messages for more details.

VeriTas-arch committed:
fix: env, markdown format and intro update

- fix environment settings add missing packages.
- fix markdown format with markdownlint.
- replace center-aligned description with left-aligned format in tables.
- update README with a better getting-started guide.
- start Chinese translation.

VeriTas-arch committed:
translation: zh-cn translation + minor fixes

- add more ignored rules to .markdownlint.json
- fix some minor formatting issues in docs/en
- complete zh-cn translations (note: there are perhaps some inaccurate translations)

VeriTas-arch committed:
minor fixes

VeriTas-arch committed:
add github action sample file

Noaaan commented:
Thanks a ton for the PR! From a glance every change here looks great.
Does the mkdocs deploy Action really need the `--force` flag?

VeriTas-arch commented:
It is just tested that the `--force` flag is not necessary. `mkdocs gh-deploy` can simply work.

VeriTas-arch commented on .github/archive/deploy.yml:
`--force` flag at line 27 is not necessary.

VeriTas-arch committed:
update workflow config

remove the `--force` flag

Noaaan commented on README.md:
````diff
@@ -2,31 +2,45 @@
 
 ```bash
 pip install mkdocs-material mkdocs-git-revision-date-localized-plugin mkdocs-video mkdocs-static-i18n[material]
+``` -->
````
Should probably just be removed

VeriTas-arch replied:
That's right

VeriTas-arch committed:
Update faq.md in `zh` folder.

Delete the unremoved English sources.

VeriTas-arch committed:
Update faq.md: Minor text error fixes

VeriTas-arch committed:
remove commented lines in README

VeriTas-arch committed:
correct the translation of fortune

VeriTas-arch committed:
fix translations for carmot staff

VeriTas-arch committed:
Fix markdown format

- Previously, the indentation of unordered list is 2, while 4 is required in order to render correctly on mkdocs.
- The problem is fixed and markdownlint.json is updated respectively.
- Some minor modifications on Chinese translations to make them more natural.

VeriTas-arch committed:
update translation for copyright

VeriTas-arch committed:
correct some translations

VeriTas-arch reviewed (commented):
I think up to now this PR is probably finished. The translations are finally checked.

Noaaan commented:
To clarify - I will review this soon™. Mainly what remains is double-checking that the chinese words for the materials are the same as the mods lang (an example being checking that Adamantite has the same name), and updating the wiki generation commands to match up with the changes from this PR

VeriTas-arch commented:
No hurry anyway :D The former comment can be simply regarded as a mark indicating major translation mistakes have been corrected. BTW, recently I found some improvable translatoins in mods lang. For example, the Chinese translation of *aegis sword* in this mod has apparently deviated that in Xenoblade, which makes its Chinese key quite confusing while the original one reamaining much clearer.

Noaaan commented:
I have never had the linguistic background to sample the Chinese version of the Xenoblade series. I also didn't know they had an official localization. Do feel free to update that if you feel like it
